packages/teleterm/src/ui/services/keyboardShortcuts/useKeyboardShortcuts.test.tsx

```diff
@@ -0,0 +1,60 @@
+import renderHook from 'design/utils/renderHook';
+import { useKeyboardShortcuts } from './useKeyboardShortcuts';
+import React from 'react';
+import AppContextProvider from 'teleterm/ui/appContextProvider';
+import { KeyboardShortcutsService } from './keyboardShortcutsService';
+import { KeyboardShortcutEventSubscriber } from './types';
+import AppContext from 'teleterm/ui/appContext';
+
+describe('teleterm/ui/services/useKeyboardShortcuts', () => {
+  it('should call handler on its event type', () => {
+    const { handler, getEventEmitter, wrapper } = getTestSetup();
```

Comment:
you do not need to use `describe` to group 1 level tests as file name and its path do the same thing (jest grouping) 

```ts
describe('teleterm/ui/services/useKeyboardShortcuts', () => {
  it('should call handler on its event type', () => {
```  

becomes

```ts
  test('handler on its event type', () => {
```  

Take a look at this guide (still in progress): 
https://gravitational.slab.com/posts/testing-guideline-proposal-wip-yuc9uvwa